packages/editor/src/store/reducer.js

```diff
@@ -810,6 +810,13 @@ export function preferences( state = PREFERENCES_DEFAULTS, action ) {
 				...state,
 				insertUsage: omitBy( state.insertUsage, ( { insert } ) => insert.ref === action.id ),
 			};
+
+		case 'ENABLE_PUBLISH_SIDEBAR':
+		case 'DISABLE_PUBLISH_SIDEBAR':
```

Comment:
I get that this is technically less code but I think it's nicer to have each action type return its own object. It's easier to grok what's happening that way–this code made sense to me but only after I looked at it for probably ten seconds; I was wondering why `'DISABLE_PUBLISH_SIDEBAR'` needed to check if the action was `'ENABLE_PUBLISH_SIDEBAR'`. I think it'd be nicer to just do:

```js
case 'ENABLE_PUBLISH_SIDEBAR':
	return {
		...state,
		isPublishSidebarEnabled: true,
	};
case 'DISABLE_PUBLISH_SIDEBAR':
	return {
		…state,
		isPublishSidebarEnabled: false,
	};
```

Reply:
Thanks for keeping us obvious.

Reply:
ee882d43c654291f10cb5e97bc8e09cfd223b14a